### PR TITLE
Updating charts & application version

### PR DESCRIPTION
**Pull request by pollier**

### CODE REVIEW

**Comment by ArgTang** [comment]
I had to change the repository to `quay.io/oauth2-proxy/oauth2-proxy` to make it work. Maybe we can change that as well in this PR?

**Comment by tlawrie** [comment]
Looks good to me.

Although I think we need some discussion as to whether we point the container image at `:latest` or we publish a new version of the chart with every release to keep updated with the released version.

@JoelSpeed do you have any preference?

**Comment by JoelSpeed** [comment]
I think we should use versioned images, otherwise if we ever make breaking changes this will break users who are running the existing releases.

In general, it's not advised to use moving tags in deployments in production so yeah, versioned tags would be my vote

**Comment by modularTaco** [comment]
I think the image repository (https://github.com/oauth2-proxy/manifests/pull/18/files#diff-d544394dd398e3408293c2dc14fe88bd77396bb56175e2c24798aa62f219125aR32) needs to be `quay.io/oauth2-proxy/oauth2-proxy` instead of `quay.io/oauth2-proxy/oauth2_proxy`.


```bash
➜ podman pull quay.io/oauth2-proxy/oauth2_proxy                                                     
Trying to pull quay.io/oauth2-proxy/oauth2_proxy:latest...
  unauthorized: access to the requested resource is not authorized
Error: Error initializing source docker://quay.io/oauth2-proxy/oauth2_proxy:latest: Error reading manifest latest in quay.io/oauth2-proxy/oauth2_proxy: unauthorized: access to the requested resource is not authorized

➜ podman pull quay.io/oauth2-proxy/oauth2-proxy
Trying to pull quay.io/oauth2-proxy/oauth2-proxy:latest...
Getting image source signatures
Copying blob 4c0d98bf9879 skipped: already exists  
Copying blob 96c61c2c0d21 [==================================>---] 9.8MiB / 10.6MiB
```

**Comment by aquam8** [comment]
Actually there's been a new version today: v7.1.0

**Comment by pierluigilenoci** [comment]
> I think the image repository ([#18 (files)](https://github.com/oauth2-proxy/manifests/pull/18/files#diff-d544394dd398e3408293c2dc14fe88bd77396bb56175e2c24798aa62f219125aR32)) needs to be `quay.io/oauth2-proxy/oauth2-proxy` instead of `quay.io/oauth2-proxy/oauth2_proxy`.

I confirm.

File: helm/oauth2-proxy/values.yaml [hunk 1]
```diff
@@ -29,8 +29,8 @@ config:
   # existingConfig: config
 
 image:
-  repository: "quay.io/pusher/oauth2_proxy"
-  tag: "v5.1.0"
+  repository: "quay.io/oauth2-proxy/oauth2_proxy"
```
Comment (pierluigilenoci) [inline helm/oauth2-proxy/values.yaml:32]: Wrong URL. The right one is `quay.io/oauth2-proxy/oauth2-proxy`
`-` instead of `_`

**Comment by pierluigilenoci** [comment]
@pollier could you please take a look?

**Comment by morganchristiansson** [comment]
Would be great it chart used latest - thanks guys was just about to fork and do my own PR for this.

**Comment by pierluigilenoci** [comment]
@pollier could you please rebase?

**Comment by erkannt** [comment]
> Would be great it chart used latest - thanks guys was just about to fork and do my own PR for this.

@morganchristiansson Pinning the version to a specific tag by default makes the chart more predictable and therefore testable.

If you prefer running `latest` you can set `.image.tag` in your `values.yaml` as you see fit.

**Comment by morganchristiansson** [comment]
Of course - I meant to pin to latest _versioned_ tag.. Maybe I should submit new PR for this as @pollier hasn't responded

**Comment by pierluigilenoci** [comment]
@pollier a rebase is needed again. 😭

**Comment by pierluigilenoci** [comment]
@erkannt @desaintmartin is there any way to move forward this PR?

**Comment by morganchristiansson** [comment]
I can raise a new PR?

**Review by desaintmartin — APPROVED** [review]
Made the needed changes.

**Comment by morganchristiansson** [comment]
LGTM

**Comment by morganchristiansson** [comment]
Build failed
```
==> Logs of container oauth2-proxy-wlm5zkorjm-9fcf68786-xwp6p
------------------------------------------------------------------------------------------------------------------------
[2021/05/13 09:47:14] [main.go:54] invalid configuration:
  cookie_secret must be 16, 24, or 32 bytes to create an AES cipher, but is 10 bytes
------------------------------------------------------------------------------------------------------------------------
<== Logs of container oauth2-proxy-wlm5zkorjm-9fcf68786-xwp6p
```

**Comment by desaintmartin** [comment]
Closing and re-creating so that I can use my fork.

**Pull request closed without merging**